byway/src/main/webapp/routepage.js

```diff
@@ -292,5 +292,34 @@ function createRecButton(rec){
   return recBtn;
 }
 
-/* exported initMap, generateRoute, placesService, map */
-/* global google, findPlace, getTripKeyFromUrl, configureTripKeyForPath */
\ No newline at end of file
+
+/**  Creates a URL link to Google Maps based on the start/end point and route
+ *   @returns {String} routeLink url containing query params for the user’s route
+ */
+function generateRouteLink(){
+    let routeLink = "https://www.google.com/maps/dir/?api=1&travelmode=driving" 
+    routeLink += "&origin=" + start.name.split(' ').join('%20') + "&destination=" + end.name.split(' ').join('%20') +"&waypoints=";
+    for(let place of route){
+      console.log(place);
+      routeLink += place.name.split(' ').join('%20') + "|";
+    }
+   return routeLink;
+}
+
+
+function sendEmail(){
```

Comment:
Add a small description of what this function does/includes

Reply:
Thanks, added a comment!